doc/content/getting-started/migrating/migrating-from-v2/migrate-using-console.md

```diff
@@ -81,7 +79,7 @@ If your end device is a part of the [LoRaWAN Device Repository]({{< ref "/integr
 
 ## Prevent the End Device from Joining {{% ttnv2 %}} Network
 
-When your end device is registered in {{% tts %}}, it is necessary to prevent it from re-joining {{% ttnv2 %}} network. 
+When your end device is registered in {{% tts %}}, it is necessary to prevent it from re-joining the {{% ttnv2 %}} network.
```

Comment:
There is already a THE in ttnv2
```suggestion
When your end device is registered in {{% tts %}}, it is necessary to prevent it from re-joining {{% ttnv2 %}} network.
```